src/configur.bat

```diff
@@ -25,36 +25,50 @@ if not exist %WATT_ROOT%\src\makefile.all goto not_set
 ::
 set     MKMAKE=..\util\mkmake.exe
 set      MKDEP=..\util\mkdep.exe
+set     MKCONF=..\util\mkconf.exe

```

Comment:
I assume you mean that `util/Makefile` should build these as needed?
For now, just add the to this PR if possible.

Reply:
Yes, will add the binaries.  It's a bit unfortunate that they have to be in the git repo.  I suppose that's mainly due to the S-Lang dependency (which not everyone might have)?

Note, I did change the utils CFLAGS to `-Os -g0`, so they should waste a bit less space.  Maybe it's an idea to upx them too?

Reply:
> I suppose that's mainly due to the S-Lang dependency (which not everyone might have)?

Right. But mkconf.c doesn't need S-Lang AFAICS. And UPX is IMHO not needed.

Reply:
So, there's not much of a reason to keep them in git right?  I did just add the mkconf binaries though.

Problem is that binaries inflate the repo size (`git clone` must download all previous versions).  And Git doesn't preserve file dates so Make can't tell if they need to be rebuilt.